frontend/src/routes/Infrastructure/Clusters/DiscoveredClusters/DiscoveryConfig/DiscoveryConfig.tsx

```diff
@@ -48,12 +49,20 @@ import {
   Provider,
 } from '../../../../../ui-components'
 import { CredentialsForm } from '../../../../Credentials/CredentialsForm'
+import { ExternalLink } from '@openshift-assisted/ui-lib/common'
```

Comment:
I'm surprised we don't have our own version of this component, but I'd prefer we don't start using this one. It includes my pet peeve, where the icon is included as part of the contents, so the underline spans the text and icon when you hover.